qiita_db/metadata_template/sample_template.py

```diff
@@ -70,6 +70,28 @@ class SampleTemplate(MetadataTemplate):
     _sample_cls = Sample
     _fp_id = convert_to_id("sample_template", "filepath_type")
     _filepath_table = 'sample_template_filepath'
+    _log_table = "sample_template_edit"
+
+    @staticmethod
```

Comment:
Lot's of code duplication in here. Possible to move to the base class and just do some smart thing to get the sql query?